### ISSUE TITLE
Expected token { at `require`

### ISSUE THREAD

**Issue by minatu2d** [demsking/image-downloader#12]
I am going to use `image-downloader` in my applications.
I tried with `require` but get followings errors,
I am quite new to `Nodejs`, please teach me how to fix it. 
Thanks

```js
var image_downloader = require('image-downloader');
```

```js
const downloader = ({ url, dest, done }) => {
                    
SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (imageDownUploader.js:1:86)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
```

**Comment by demsking** [comment]
This is a `destructuring` feature, available since the `v6.4.0` of NodeJS.
You have to upgrade your NodeJS version to use this feature.

**Comment by minatu2d** [comment]
Thanks for your answer. 
I will try it.
